releases/release-1.27/release-team.md

```diff
@@ -0,0 +1,10 @@
+# Kubernetes 1.27 Release Team
+
+| **Team/Role** | **Lead Name** (**GitHub / Slack ID**) | **Shadow Name(s) (GitHub / Slack ID)** |
+|----------|----------------------------------|----------------------------------------|
+| Release Team Lead | Xander Grzywinski ([@salaxander](https://github.com/salaxander) / Slack: `@Xander`) | Kat Cosgrove ([@katcosgrove](https://github.com/katcosgrove) / Slack: `@katcosgrove`) |
+| Emeritus Adviser | James Laverack ([@jameslaverack](https://github.com/jameslaverack)) / Slack: `@james.laverack` | |
+
+Review the [Release Managers page](https://github.com/kubernetes/website/blob/main/content/en/releases/release-managers.md) for up-to-date contact information on Release Engineering personnel.
+
+The schedule for all patch releases can be found at [Patch Releases page](https://github.com/kubernetes/website/blob/main/content/en/releases/patch-releases.md). It will be updated to include 1.26, once the 1.26 release cycle concludes.
```

Comment:
```suggestion
The schedule for all patch releases can be found at [Patch Releases page](https://github.com/kubernetes/website/blob/main/content/en/releases/patch-releases.md). It will be updated to include 1.27, once the 1.27 release cycle concludes.
```